Feature/post message in group

Celoka opened the pull request.

Celoka committed:
Merge pull request #3 from Celoka/routes

Routes

Celoka committed:
Merge pull request #4 from Celoka/routes

Code refactor

Celoka committed:
Merge pull request #5 from Celoka/routes

Edited the addUserToGroup route to display the name of group members

Celoka committed:
Merge pull request #7 from Celoka/routes

Changed firebase configuration key

Celoka committed:
Merge pull request #8 from Celoka/routes

config change

Celoka committed:
Create README.md

Celoka committed:
Update README.md

Celoka committed:
Update README.md

Celoka committed:
Update README.md

Celoka committed:
Merge pull request #9 from Celoka/routes

Code Refactor

Celoka committed:
API unit test

Celoka committed:
Merge pull request #11 from Celoka/routes

API unit test

Celoka committed:
API Unit Test: Server and test file.

Celoka committed:
Merge pull request #12 from Celoka/routes

API Unit Test: Server and Test file.

Celoka committed:
API Route and configuration files

Celoka committed:
Merge pull request #13 from Celoka/routes

API Route and configuration files

Celoka committed:
Delete server.js

Celoka committed:
Delete config.js

Celoka committed:
Delete routes.js

Celoka committed:
Update README.md

Celoka committed:
Merge pull request #14 from Celoka/develop1

Develop1

Celoka committed:
Merge pull request #15 from Celoka/routes

Merge pull request #14 from Celoka/develop1

Celoka committed:
Folder restructure

Celoka committed:
Folder restructuring

Celoka committed:
Merge pull request #16 from Celoka/routes

Routes

Celoka committed:
Wrote more test for routes

Celoka committed:
Modified the file: test.js is now served by server.js. Deleted route-testserver.js

Celoka committed:
Merge pull request #17 from Celoka/routes

Wrote more test for routes

Celoka committed:
Created new files for post-it app

Celoka committed:
Merge pull request #18 from Celoka/routes

Created new files for post-it app

Celoka committed:
General code refactor

Celoka committed:
Merge pull request #19 from Celoka/routes

General code refactor

Celoka committed:
Test.js: Code Refactor

Celoka committed:
Merge pull request #20 from Celoka/routes

Test.js: Code Refactor

Celoka committed:
Folder restructure and creation of new components

Celoka committed:
Merge pull request #21 from Celoka/routes

Folder restructure and creation of new components

Celoka committed:
Delete appstore.js

Celoka committed:
Merge pull request #22 from Celoka/routes

Delete appstore.js

Celoka committed:
Deletion of one public folder in Client Directory

Celoka committed:
Merge pull request #23 from Celoka/routes

Deletion of one public folder in Client Directory

Celoka committed:
Edited all the components and modified the routes.

Celoka committed:
Merge pull request #24 from Celoka/routes

Edited all the components and modified the routes.

Celoka committed:
Modified LoginPAge and SignUpPage

Celoka committed:
Merge pull request #25 from Celoka/routes

Modified LoginPage and SignUpPage

Celoka committed:
App component Test

Celoka committed:
Merge pull request #26 from Celoka/routes

App component Test

Celoka committed:
Code Refactor.

Celoka committed:
Merge pull request #27 from Celoka/routes

Code Refactor.

Celoka committed:
Code Refactor: Changed all the application components.

Celoka committed:
Merge pull request #28 from Celoka/routes

Code Refactor: Changed all the application components.

Celoka committed:
Modified Addmember, Creategroup and Footer components. Deleted app folder in dist

Celoka committed:
Merge pull request #29 from Celoka/routes

Modified Addmember, Creategroup and Footer components. Deleted app fo…

Celoka committed:
Delete HomePage.jsx

Celoka committed:
File modification

Celoka committed:
Deleted Homepage.jsx from Components folder

Celoka committed:
Delete HomePage.jsx

Celoka committed:
Delete HomePage.jsx

Celoka committed:
Code refcator

Celoka committed:
Resolved Conflict

Celoka committed:
Homepage.jsx Delete

Celoka committed:
Delete HomePage.jsx

Celoka committed:
Delete HomePage.jsx

Celoka committed:
Line Edit in Footer

Celoka committed:
Footer.jsx Edit

Celoka committed:
Merge pull request #32 from Celoka/routes

Routes

Celoka committed:
Code refactor

Celoka committed:
Merge pull request #33 from Celoka/routes

Code refactor

Celoka committed:
Modification of server, webpack server and package.json

Celoka committed:
Merge pull request #34 from Celoka/routes

Modification of server.js, webpack server and package.json

Celoka committed:
React hot reloader installed in dependecies

Celoka committed:
Refactored webpack config

Celoka committed:
Merge pull request #35 from Celoka/routes

React hot reloader installed in dependencies

Celoka committed:
Added JsDoc comment

Celoka committed:
Merge pull request #36 from Celoka/routes

Added JsDoc comment

Celoka committed:
Component Name change Sign Up And SignIn

Celoka committed:
Merge pull request #37 from Celoka/routes

Component Name change Sign Up And SignIn

Celoka committed:
Removed a Route duplicate for home component

Celoka committed:
Merge pull request #38 from Celoka/routes

Removed a Route duplicate for home component

Celoka committed:
Git ignored dist folder and created server folder for server and routes

Celoka committed:
Merge pull request #39 from Celoka/routes

Git ignored dist folder and created server folder for server and routes

Celoka committed:
feature: integrate  .env, create controllers for routes, refactor api.

-refactor route.
-create controllers for users and groups.
-create a donenv for firebase config key.

Celoka committed:
Merge pull request #40 from Celoka/routes

feature: integrate  .env, create controllers for routes, refactor api.

Celoka committed:
Route refactor

Celoka committed:
Merge pull request #41 from Celoka/routes

Route refactor

Celoka committed:
Merge pull request #41 from Celoka/routes

Route refactor

Reset Password

feature: implement post message to group

- create messageboard component
- create actions, dispatch actions, and message store

[Deliver #4]

feature(message): implement retrieve message sent to group

- create child component for message board and pass props to it
- pass props from dashboard to message board
- install nexmo and nodemailer
- refactor post message and get message controllers

Create LICENSE

Update README.md

feature(add user): implement add user to group

- add JSdoc to all the required functions

chore(test):  write server side test

- write side test using chai to test the quality of the code written.
- installed all the test dependencies required.

    chore(test):  write server side test

    - deleted auto generated files

chore(test):  write server side test

- modified travis,yml

chore(test):  write server side test

- Modify travis.yml to pass test and obtain coverage.

chore(test):  write server side test

- Comment some failing test to check if there would be a build on travis.

Celoka committed:
Reset Password

Celoka committed:
Merge pull request #42 from Celoka/routes

Reset Password

Celoka committed:
feature: implement post message to group

- create messageboard component
- create actions, dispatch actions, and message store

[Deliver #4]

efalayi commented on client/src/components/ResetPassword.jsx:
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```
This probably has to do with your ESLint config.

efalayi commented on client/src/components/ResetPassword.jsx:
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import AppActions from '../actions/AppActions';
+import Header from '../components/Navbar.jsx';
```
Remove the Header import since it is not being used.

efalayi commented on client/src/components/Signup.jsx:
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import AppStore from '../stores/AppStore';
```
Same with AppStore

efalayi commented on client/src/components/Signup.jsx:
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import AppStore from '../stores/AppStore';
+import Header from '../components/Navbar.jsx';
+import AppActions from '../actions/AppActions';
```
Include the file extension for AppActions. It should probably be a '.js'

efalayi commented:
`Summary`
- Fix your eslint errors
- Remove unused variables and comments
- Add JSDoc. Documenting your code helps other developers understand your codebase
- I noticed you have some merge conflicts. You also try to fix these.

Celoka committed:
feature(message): implement retrieve message sent to group

- create child component for message board and pass props to it
- pass props from dashboard to message board
- install nexmo and nodemailer
- refactor post message and get message controllers

Celoka committed:
Merge pull request #45 from Celoka/feature/retrieve-group-message

Feature/retrieve group message

Celoka committed:
Create LICENSE

Celoka committed:
Update README.md

Celoka committed:
feature(add user): implement add user to group

- add JSdoc to all the required functions

Celoka committed:
Merge pull request #47 from Celoka/feature/add-user-to-group

feature/add user to group

Celoka committed:
Merge pull request #48 from Celoka/routes

feature/ add user to group

Celoka committed:
chore(test):  write server side test

- write side test using chai to test the quality of the code written.
- installed all the test dependencies required.

Celoka committed:
chore(test):  write server side test

    - deleted auto generated files

Celoka committed:
Merge pull request #49 from Celoka/chore/server-side-test

Chore/server side test

Celoka committed:
chore(test):  write server side test

- modified travis,yml

Celoka committed:
Merge pull request #50 from Celoka/chore/server-side-test

chore(test):  write server side test

Celoka committed:
chore(test):  write server side test

- Modify travis.yml to pass test and obtain coverage.

Celoka committed:
Merge pull request #51 from Celoka/chore/server-side-test

chore(test):  write server side test

Celoka committed:
chore(test):  write server side test

- Comment some failing test to check if there would be a build on travis.

Celoka committed:
Merge pull request #52 from Celoka/chore/server-side-test

chore(test):  write server side test

Celoka committed:
chore(test):  write server side test

- Attach travis and coverall badges to indicate build and coverage respectively

Celoka committed:
Merge pull request #53 from Celoka/chore/server-side-test

Chore/server side test

Celoka committed:
feature: implement post message to group

- Resolve bug.

Celoka committed:
feature: implement post message to group

- Change status code to reflect the right status.

Celoka committed:
feature(add-user): implement add user to group

- refactor add user route
- re-configure webpack to allow for google sign in implementation
- create a dotenv file to add secret key to the environment variable
- reconfigure app store and action to fix bug
- catch errors at the view
- set up front end test with jest and enzyme

[Delivers  #001 ]

Celoka committed:
feature(notification): implement message notification

- use nodemailer and nexmo to implement sms notification

[Delivers 001]

Celoka committed:
bug(flight): fix sms notification bug

 -refactor code for nexmo to send sms notification

[Delivers 001]

Celoka committed:
bug(flight): fix sms notification bug

 - refactor code in dashboard component and group component
- fix post message bug
- remove all console log

[Delivers 001]

Celoka committed:
feature(add-member): implement add registered member to group

- complete feature implementation
- change the view interface of the app

[Finishes ]

Celoka committed:
feature(cursor-pointer): implement cursor pointer

- add cursor pointer for on hover and click group

[Finishes ]

Celoka committed:
refactor(): ensure appriopriate component type are used

- refcatoring component code to change some stateful components to stateless

[Finishes]

Celoka committed:
refactor(env): ensure secret keys are put in environment variable

- refcatoring component code to change some stateful components to stateless

 [Finishes]

Celoka committed:
test(componen): write test for all components and actions

- write test for some component
- set up test for actions
- create webpack for production

[Finishes]

Celoka committed:
feature(post): implement post message to group

- Resolve bug after rebase off staging

[Finishes ]
